FluentTc.Tests/AcceptanceTests.cs

```diff
@@ -1599,5 +1599,99 @@ public void SetBuildName(string name)
                     HttpContentTypes.TextPlain, "/app/rest/buildTypes/id:BuildId/name", string.Empty))
                         .MustHaveHappened(Repeated.Exactly.Once);
         }
+
+
+
+
+        [Test]
+        public void CreateVcsRoot()
+        {
+            // Arrange
+            var teamCityCaller = CreateTeamCityCaller();
+
+            var connectedTc = new RemoteTc().Connect(_ => _.AsGuest(), teamCityCaller);
+            var project = connectedTc.GetProjectById("ProjectId");
+
+            // Act
+            var vcsRoot = connectedTc.CreateVcsRoot(project, __ => __
+                .AgentCleanFilePolicy(AgentCleanFilePolicy.AllIgnoredUntrackedFiles)
+                .AgentCleanPolicy(AgentCleanPolicy.Always)
+                .AuthMethod(AuthMethod.Anonymous)
+                .Branch("refs/head/develop")
+                .BranchSpec("+:refs/head/feature/*")
```

Comment:
It would be super cool if we could add a builder for branch spec. But maybe this is too much